builders/get-started/networks/moonbeam-dev.md

```diff
@@ -51,15 +51,15 @@ Using Docker enables you to spin up a node in a matter of seconds. Once you have
         ```bash
```

Comment:
Just checking is there a reason we don't have the RPC external flag on the Ubuntu command?

Reply:
oversight. i'll add it! thanks! 🎣